docs/schema/localizeddisplayname-element-in-localizeddisplaynames-bdcmetadata-schema.md

```diff
@@ -59,24 +59,10 @@ None.
 
 ### Parent elements
 
-<table>
-<colgroup>
-<col width="50%" />
-<col width="50%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>Element</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The **LocalizedDisplayNames** element that contains this **LocalizedDisplayName**.</p></td>
-</tr>
-</tbody>
-</table>
+  
+| Element | Description |
+| --- | --- |
+| [LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)](localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md) | The \*\*LocalizedDisplayNames\*\* element that contains this \*\*LocalizedDisplayName\*\*. |
```

Comment:
Let's delete the slashes around the asterisks for "LocalizedDisplayNames."  So it looks like:
`| [LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)](localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md) | The **LocalizedDisplayNames** element that contains this **LocalizedDisplayName**. |`